test/run-php-linter.sh

```diff
@@ -52,8 +52,8 @@ vendor/bin/phpcs --standard=test/LorisCS.xml --extensions=php,inc \
     php/ \
     htdocs/ \
     modules/ \
+    "test/integrationtests/" \
```

Comment:
The variable "test_list" above doesn't appear to be used anymore and should be deleted.

Reply:
@johnsaigle Any reason why PHPCS is not run on the test_list anymore?

Reply:
You changed it to run on the entire directory that had the files in the list.

Reply:
Oh, missed that. You're right then.